Modules/TRD/include/TRD/LinkDef.h

```diff
@@ -0,0 +1,8 @@
+#ifdef __CLING__
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+#pragma link C++ class o2::quality_control_modules::trd::TRDDigitQcTask+;
```

Comment:
I would rename this class to `o2::quality_control_modules::trd::DigitsTask`. This seems to be more in line with TOF and TPC naming conventions.